chain/client/tests/bug_repros.rs

```diff
@@ -159,7 +157,7 @@ fn test_sync_from_achival_node() {
             false,
             false,
             epoch_length,
-            false,
+            true,
```

Comment:
what is the reason of turning Doomslug on in the test?

Reply:
I was experimenting. It is not needed.